my-code/movie-search/src/Containers/MovieDetails/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { fetchMovie } from '../../Actions';
+
+class MovieDetails extends Component {
+  componentDidMount() {
+    const { imdbID } = this.props.match.params;
+    this.props.fetchMovie(imdbID);
+
+    console.log(this.props);
+  }
+
+  render() {
+    const movie = this.props.chosenMovie;
```

Comment:
you could use ES6 destructuring: https://codeburst.io/es6-destructuring-the-complete-guide-7f842d08b98f?gi=d93f8df8d0a1

```jsx
const { chosenMovie: movie } = this.props;
```

Reply:
This will be a must-read for me. I've only started understanding when I can destructure.